app/controllers/patients_controller.rb

```diff
@@ -18,10 +19,33 @@ def index
     @patients = paginate(@patients)
   end
 
+  # This controller / page does not have unit-tests since it's potentially throwaway work.
+  # If we decide to continue using this, we should invest in testing it.
+  #
+  # Because of its throwaway nature, it intentionally piggybacks on the
+  # view_overdue_list permission to avoid making a hyper-specific permission for this change.
+  def lookup
+    set_page
+    set_per_page
+    set_facility_id
+    authorize([:overdue_list, Patient])
+
+    if current_facility
+      @patients =
+        paginate(policy_scope([:overdue_list, Patient])
```

Comment:
Does this policy scope point to the new policy scope you've created inside `OverdueList::PatientPolicy`?

Reply:
It is, it's some pundit black-magic that I don't quite understand, but that's how it's supposed to work.

Reply:
It magically picks up the `class` and method name and looks it up in the `Policy` class, probably some metaprogramming magic behind-the-scenes.

Reply:
Hehe I'm fine with the magic, just making sure it was grabbing the scope I _thought_ it was. 👍